src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -15,14 +16,19 @@ public class SwaggerMiddleware
         private readonly RequestDelegate _next;
         private readonly SwaggerOptions _options;
         private readonly TemplateMatcher _requestMatcher;
+        private readonly ISwaggerDocumentSerializer _swaggerDocumentSerializer;
 
         public SwaggerMiddleware(
             RequestDelegate next,
-            SwaggerOptions options)
+            SwaggerOptions options,
+            IServiceProvider serviceProvider)
```

Comment:
This is a binary-breaking change, so we could only take this as part of a major release (i.e. v7).

Reply:
Would it also be breaking if we add it as a second constructor? And linking the previous constructor to this one?

Reply:
No that wouldn't be breaking, provided that DI knows which constructor to use.

Reply:
Quick question, should I "resolve" conversations myself when I've handled them (even if I didn't push yet) or leave that to the original commenter?